frontend/Views/SingleDiscussion/index.js

```diff
@@ -173,6 +195,7 @@ class SingleDiscussion extends Component {
           allowDelete={allowDelete}
           deletingDiscussion={deletingDiscussion}
           deleteAction={this.deleteDiscussion.bind(this)}
+          mapViewChildComponent={this.renderMapViewChildComponent()}
```

Comment:
Is this supposed to invoke? Seems like it should be just a function reference

Reply:
it works either way, React accepts both functions that return components/elements and elements themselves as props. I think it's a preference thing to have it one way vs the other. If you want, I can change this to a func and call it in the child!